instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/utils/http/MockDirectionsRequestHandler.kt

```diff
@@ -40,4 +40,13 @@ data class MockDirectionsRequestHandler(
 
         return this.joinToString(";") { "${it.longitude()},${it.latitude()}" }
     }
+
+    override fun toString(): String {
+        return "MockDirectionsRequestHandler(" +
+            "profile='$profile', " +
+            "expectedCoordinates=$expectedCoordinates, " +
+            "relaxedExpectedCoordinates=$relaxedExpectedCoordinates, " +
+            "jsonResponse='$jsonResponse'" +
+            ")"
+    }
```

Comment:
Detected some [instrumentation test failures](https://console.firebase.google.com/project/mapbox-navigation-android/testlab/histories/bh.54eb2dd57e959221/matrices/8822947192342881303/executions/bs.964fa309fa0c4d7c/testcases/11/logs).

Adding this `toString` method to get more useful crash log messages.
Vanilla `toString` method trimms `expectedCoordinates` and `relaxedExpectedCoordinates` values.

```
06-16 19:52:06.176: D/Mapbox(26275): --------- beginning of crash
06-16 19:52:06.184: E/AndroidRuntime(26275): FATAL EXCEPTION: MockWebServer TaskRunner
06-16 19:52:06.184: E/AndroidRuntime(26275): Process: com.mapbox.navigation.instrumentation_tests, PID: 26275
06-16 19:52:06.184: E/AndroidRuntime(26275): java.lang.Error: Request url:
06-16 19:52:06.184: E/AndroidRuntime(26275): /directions/v5/mapbox/driving-traffic/-77.3271476,38.948794;-121.480279,38.57674?access_token=REDACTED&geometries=polyline6&alternatives=true&overview=full&steps=true&avoid_maneuver_radius=10.978466033935547&bearings=13.3707848%2C90%3B&layers=%3B&continue_straight=true&annotations=congestion_numeric%2Cmaxspeed%2Cclosure%2Cspeed%2Cduration%2Cdistance&roundabout_exits=true&voice_instructions=true&banner_instructions=true&enable_refresh=true&sku=10atDJy3NvJMo
06-16 19:52:06.184: E/AndroidRuntime(26275): is not handled.
06-16 19:52:06.184: E/AndroidRuntime(26275): Available handlers:
06-16 19:52:06.184: E/AndroidRuntime(26275): MockDirectionsRequestHandler(profile=driving-traffic, jsonResponse={"routes":[{"weight_typical":366.114,"duration_typical":260.004,"weight_name":"auto","weight":358.967,"duration":254.169,"distance":2045.35,"legs":[{"admins":[{"iso_3166_1_alpha3":"USA","iso_3166_1":"US"}],"incidents":[{"length":50,"affected_road_names":["9th Street"],"id":"11589180127444257","type":"lane_restriction","congestion":{"value":20},"description":"Rummelsburger Landstrasse: Bauarbeiten um Fritz-K\u00f6nig-Weg","long_description":"Fahrbahnverengung von auf eine Fahrspur wegen Bauarbeiten auf der Rummelsburger Landstrasse in Richtung S\u00fcden zwischen Rummelsburger Landstrasse und Fritz-K\u00f6nig-Weg.","impact":"low","alertc_codes":[743],"geometry_index_start":3,"geometry_index_end":8,"creation_time":"2022-05-11T14:10:36Z","start_time":"2022-04-29T13:08:25Z","end_time":"2022-05-30T21:59:00Z","iso_3166_1_alpha2":"US","iso_3166_1_alpha3":"USA","lanes_blocked":[]}],"annotation":{"maxspeed":[{"unknown":true},{"unknown":true},{"unknown":true},{"unknown":true},{"unknown":true},{"unknown":true},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"unknown":true},{"unknown":true},{"unknown":true},{"unknown":true},{"unknown":true}],"congestion":["unknown","unknown","unknown","unknown","moderate","moderate","unknown","unknown","unknown","unknown","unknown","low","low","low","unknown","unknown","unknown","low","low","low","unknown","unknown","low","low","low","unknown","unknown","unknown","unknown","unknown","low","low","low","low","low","low","unknown","unknown","unknown","unknown","low","low","low","low","unknown","unknown","unknown","unknown","low"],"speed":[4.2,4.2,10.6,10.6,9.4,9.4,6.1,9.7,9.7,9.7,9.7,7.5,7.5,7.5,11.4,11.4,11.4,12.8,12.8,12.8,6.9,6.9,10.3,5.8,10,10.8,11.7,11.7,11.7,8.9,10.3,10.3,9.7,11.1,11.1,5.6,11.9,11.9,13.9,13.9,13.6,13.6,16.1,11.7,9.7,9.4,9.7,9.7,9.4],"distance":[27.2,8.9,13.3,11.4,10.7,109.9,121.8,6.9,4,100.3,10.6,11.2,101.6,10.2,10.1,126.8,11,9.3,102,11.1,11.5,110.5,63.9,58.7,64.3,48.2,18.8,3.7,14.6,113.9,19.6,45.6,64.3,26.4,38.4,64.4,48.6,31.5,9.7,30.5,50.8,13.8,61.4,122.1,34.1,29.9,7.8,8,9.2],"duration":[6.518,2.144,1.258,1.077,1.134,11.638,19.931,0.709,0.409,10.317,1.093,1.5,13.541,1.362,0.89,11.132,0.969,0.727,7.985,0.87,1.654,15.913,6.217,10.058,6.432,4.449,1.614,0.3
06-16 19:52:06.186: D/Mapbox(26275): [nav-native]: Entering void mapbox::navigation::history::Recorder::addRecordWithPayload(mapbox::navigation::history::RecordPayload)
06-16 19:52:06.186: D/Mapbox(26275): [nav-native]: Exiting void mapbox::navigation::history::Recorder::addRecordWithPayload(mapbox::navigation::history::RecordPayload)
06-16 19:52:06.186: D/Mapbox(26275): [nav-native]: Entering void mapbox::navigation::history::Recorder::addRecordWithPayload(mapbox::navigation::history::RecordPayload)
06-16 19:52:06.187: D/Mapbox(26275): [nav-native]: Exiting void mapbox::navigation::history::Recorder::addRecordWithPayload(mapbox::navigation::history::RecordPayload)
06-16 19:52:06.187: D/Error(26275): ERR: exClass=java.lang.Error
06-16 19:52:06.187: D/Error(26275): ERR: exMsg=Request url:
06-16 19:52:06.187: D/Error(26275): /directions/v5/mapbox/driving-traffic/-77.3271476,38.948794;-121.480279,38.57674?access_token=REDACTED&geometries=polyline6&alternatives=true&overview=full&steps=true&avoid_maneuver_radius=10.978466033935547&bearings=13.3707848%2C90%3B&layers=%3B&continue_straight=true&annotations=congestion_numeric%2Cmaxspeed%2Cclosure%2Cspeed%2Cduration%2Cdistance&roundabout_exits=true&voice_instructions=true&banner_instructions=true&enable_refresh=true&sku=10atDJy3NvJMo
06-16 19:52:06.187: D/Error(26275): is not handled.
06-16 19:52:06.187: D/Error(26275): Available handlers:
06-16 19:52:06.187: D/Error(26275): MockDirectionsRequestHandler(profile=driving-traffic, jsonResponse={"routes":[{"weight_typical":366.114,"duration_typical":260.004,"weight_name":"auto","weight":358.967,"duration":254.169,"distance":2045.35,"legs":[{"admins":[{"iso_3166_1_alpha3":"USA","iso_3166_1":"US"}],"incidents":[{"length":50,"affected_road_names":["9th Street"],"id":"11589180127444257","type":"lane_restriction","congestion":{"value":20},"description":"Rummelsburger Landstrasse: Bauarbeiten um Fritz-K\u00f6nig-Weg","long_description":"Fahrbahnverengung von auf eine Fahrspur wegen Bauarbeiten auf der Rummelsburger Landstrasse in Richtung S\u00fcden zwischen Rummelsburger Landstrasse und Fritz-K\u00f6nig-Weg.","impact":"low","alertc_codes":[743],"geometry_index_start":3,"geometry_index_end":8,"creation_time":"2022-05-11T14:10:36Z","start_time":"2022-04-29T13:08:25Z","end_time":"2022-05-30T21:59:00Z","iso_3166_1_alpha2":"US","iso_3166_1_alpha3":"USA","lanes_blocked":[]}],"annotation":{"maxspeed":[{"unknown":true},{"unknown":true},{"unknown":true},{"unknown":true},{"unknown":true},{"unknown":true},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"unknown":true},{"unknown":true},{"unknown":true},{"unknown":true},{"unknown":true}],"congestion":["unknown","unknown","unknown","unknown","moderate","moderate","unknown","unknown","unknown","unknown","unknown","low","low","low","unknown","unknown","unknown","low","low","low","unknown","unknown","low","low","low","unknown","unknown","unknown","unknown","unknown","low","low","low","low","low","low","unknown","unknown","unknown","unknown","low","low","low","low","unknown","unknown","unknown","unknown","low"],"speed":[4.2,4.2,10.6,10.6,9.4,9.4,6.1,9.7,9.7,9.7,9.7,7.5,7.5,7.5,11.4,11.4,11.4,12.8,12.8,12.8,6.9,6.9,10.3,5.8,10,10.8,11.7,11.7,11.7,8.9,10.3,10.3,9.7,11.1,11.1,5.6,11.9,11.9,13.9,13.9,13.6,13.6,16.1,11.7,9.7,9.4,9.7,9.7,9.4],"distance":[27.2,8.9,13.3,11.4,10.7,109.9,121.8,6.9,4,100.3,10.6,11.2,101.6,10.2,10.1,126.8,11,9.3,102,11.1,11.5,110.5,63.9,58.7,64.3,48.2,18.8,3.7,14.6,113.9,19.6,45.6,64.3,26.4,38.4,64.4,48.6,31.5,9.7,30.5,50.8,13.8,61.4,122.1,34.1,29.9,7.8,8,9.2],"duration":[6.518,2.144,1.258,1.077,1.134,11.638,19.931,0.709,0.409,10.317,1.093,1.5,13.541,1.362,0.89,11.132,0.969,0.727,7.985,0.87,1.654,15.913,6.217,10.058,6.432,4.449,1.614,0.32,1.252,12.814,1.903,4.442,6.616,2.379,3.459,11.6,4.068,2.64,0.695,2.199,3.735,1.013,3.81,10.466,3.505,3.165,0.801,0.826,0
06-16 19:52:06.187: D/Error(26275): ERR: file=MockWebServerRule.kt
06-16 19:52:06.187: D/Error(26275): ERR: class=com.mapbox.navigation.testing.ui.http.MockWebServerRule$1
06-16 19:52:06.187: D/Error(26275): ERR: method=dispatch line=46
06-16 19:52:06.188: D/Error(26275): ERR: stack=java.lang.Error: Request url:
06-16 19:52:06.188: D/Error(26275): /directions/v5/mapbox/driving-traffic/-77.3271476,38.948794;-121.480279,38.57674?access_token=REDACTED&geometries=polyline6&alternatives=true&overview=full&steps=true&avoid_maneuver_radius=10.978466033935547&bearings=13.3707848%2C90%3B&layers=%3B&continue_straight=true&annotations=congestion_numeric%2Cmaxspeed%2Cclosure%2Cspeed%2Cduration%2Cdistance&roundabout_exits=true&voice_instructions=true&banner_instructions=true&enable_refresh=true&sku=10atDJy3NvJMo
06-16 19:52:06.188: D/Error(26275): is not handled.
06-16 19:52:06.188: D/Error(26275): Available handlers:
06-16 19:52:06.188: D/Error(26275): MockDirectionsRequestHandler(profile=driving-traffic, jsonResponse={"routes":[{"weight_typical":366.114,"duration_typical":260.004,"weight_name":"auto","weight":358.967,"duration":254.169,"distance":2045.35,"legs":[{"admins":[{"iso_3166_1_alpha3":"USA","iso_3166_1":"US"}],"incidents":[{"length":50,"affected_road_names":["9th Street"],"id":"11589180127444257","type":"lane_restriction","congestion":{"value":20},"description":"Rummelsburger Landstrasse: Bauarbeiten um Fritz-K\u00f6nig-Weg","long_description":"Fahrbahnverengung von auf eine Fahrspur wegen Bauarbeiten auf der Rummelsburger Landstrasse in Richtung S\u00fcden zwischen Rummelsburger Landstrasse und Fritz-K\u00f6nig-Weg.","impact":"low","alertc_codes":[743],"geometry_index_start":3,"geometry_index_end":8,"creation_time":"2022-05-11T14:10:36Z","start_time":"2022-04-29T13:08:25Z","end_time":"2022-05-30T21:59:00Z","iso_3166_1_alpha2":"US","iso_3166_1_alpha3":"USA","lanes_blocked":[]}],"annotation":{"maxspeed":[{"unknown":true},{"unknown":true},{"unknown":true},{"unknown":true},{"unknown":true},{"unknown":true},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"unknown":true},{"unknown":true},{"unknown":true},{"unknown":true},{"unknown":true}],"congestion":["unknown","unknown","unknown","unknown","moderate","moderate","unknown","unknown","unknown","unknown","unknown","low","low","low","unknown","unknown","unknown","low","low","low","unknown","unknown","low","low","low","unknown","unknown","unknown","unknown","unknown","low","low","low","low","low","low","unknown","unknown","unknown","unknown","low","low","low","low","unknown","unknown","unknown","unknown","low"],"speed":[4.2,4.2,10.6,10.6,9.4,9.4,6.1,9.7,9.7,9.7,9.7,7.5,7.5,7.5,11.4,11.4,11.4,12.8,12.8,12.8,6.9,6.9,10.3,5.8,10,10.8,11.7,11.7,11.7,8.9,10.3,10.3,9.7,11.1,11.1,5.6,11.9,11.9,13.9,13.9,13.6,13.6,16.1,11.7,9.7,9.4,9.7,9.7,9.4],"distance":[27.2,8.9,13.3,11.4,10.7,109.9,121.8,6.9,4,100.3,10.6,11.2,101.6,10.2,10.1,126.8,11,9.3,102,11.1,11.5,110.5,63.9,58.7,64.3,48.2,18.8,3.7,14.6,113.9,19.6,45.6,64.3,26.4,38.4,64.4,48.6,31.5,9.7,30.5,50.8,13.8,61.4,122.1,34.1,29.9,7.8,8,9.2],"duration":[6.518,2.144,1.258,1.077,1.134,11.638,19.931,0.709,0.409,10.317,1.093,1.5,13.541,1.362,0.89,11.132,0.969,0.727,7.985,0.87,1.654,15.913,6.217,10.058,6.432,4.449,1.614,0.32,1.252,12.814,1.903,4.442,6.616,2.379,3.459,11.6,4.068,2.64,0.695,2.199,3.735,1.013,3.81,10.466,3.505,3.
06-16 19:52:06.188: D/Error(26275): ERR: TOTAL BYTES WRITTEN: 171216
06-16 19:52:06.193: W/ActivityManager(874): Error in app com.mapbox.navigation.instrumentation_tests running instrumentation ComponentInfo{com.mapbox.navigation.instrumentation_tests.test/androidx.test.runner.AndroidJUnitRunner}:
06-16 19:52:06.193: W/ActivityManager(874):   java.lang.Error
06-16 19:52:06.194: W/ActivityManager(874):   java.lang.Error: Request url:
06-16 19:52:06.194: W/ActivityManager(874): /directions/v5/mapbox/driving-traffic/-77.3271476,38.948794;-121.480279,38.57674?access_token=REDACTED&geometries=polyline6&alternatives=true&overview=full&steps=true&avoid_maneuver_radius=10.978466033935547&bearings=13.3707848%2C90%3B&layers=%3B&continue_straight=true&annotations=congestion_numeric%2Cmaxspeed%2Cclosure%2Cspeed%2Cduration%2Cdistance&roundabout_exits=true&voice_instructions=true&banner_instructions=true&enable_refresh=true&sku=10atDJy3NvJMo
06-16 19:52:06.194: W/ActivityManager(874): is not handled.
06-16 19:52:06.194: W/ActivityManager(874): Available handlers:
06-16 19:52:06.194: W/ActivityManager(874): MockDirectionsRequestHandler(profile=driving-traffic, jsonResponse={"routes":[{"weight_typical":366.114,"duration_typical":260.004,"weight_name":"auto","weight":358.967,"duration":254.169,"distance":2045.35,"legs":[{"admins":[{"iso_3166_1_alpha3":"USA","iso_3166_1":"US"}],"incidents":[{"length":50,"affected_road_names":["9th Street"],"id":"11589180127444257","type":"lane_restriction","congestion":{"value":20},"description":"Rummelsburger Landstrasse: Bauarbeiten um Fritz-K\u00f6nig-Weg","long_description":"Fahrbahnverengung von auf eine Fahrspur wegen Bauarbeiten auf der Rummelsburger Landstrasse in Richtung S\u00fcden zwischen Rummelsburger Landstrasse und Fritz-K\u00f6nig-Weg.","impact":"low","alertc_codes":[743],"geometry_index_start":3,"geometry_index_end":8,"creation_time":"2022-05-11T14:10:36Z","start_time":"2022-04-29T13:08:25Z","end_time":"2022-05-30T21:59:00Z","iso_3166_1_alpha2":"US","iso_3166_1_alpha3":"USA","lanes_blocked":[]}],"annotation":{"maxspeed":[{"unknown":true},{"unknown":true},{"unknown":true},{"unknown":true},{"unknown":true},{"unknown":true},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":40,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"speed":48,"unit":"km\/h"},{"unknown":true},{"unknown":true},{"unknown":true},{"unknown":true},{"unknown":true}],"congestion":["unknown","unknown","unknown","unknown","moderate","moderate","unknown","unknown","unknown","unknown","unknown","low","low","low","unknown","unknown","unknown","low","low","low","unknown","unknown","low","low","low","unknown","unknown","unknown","unknown","unknown","low","low","low","low","low","low","unknown","unknown","unknown","unknown","low","low","low","low","unknown","unknown","unknown","unknown","low"],"speed":[4.2,4.2,10.6,10.6,9.4,9.4,6.1,9.7,9.7,9.7,9.7,7.5,7.5,7.5,11.4,11.4,11.4,12.8,12.8,12.8,6.9,6.9,10.3,5.8,10,10.8,11.7,11.7,11.7,8.9,10.3,10.3,9.7,11.1,11.1,5.6,11.9,11.9,13.9,13.9,13.6,13.6,16.1,11.7,9.7,9.4,9.7,9.7,9.4],"distance":[27.2,8.9,13.3,11.4,10.7,109.9,121.8,6.9,4,100.3,10.6,11.2,101.6,10.2,10.1,126.8,11,9.3,102,11.1,11.5,110.5,63.9,58.7,64.3,48.2,18.8,3.7,14.6,113.9,19.6,45.6,64.3,26.4,38.4,64.4,48.6,31.5,9.7,30.5,50.8,13.8,61.4,122.1,34.1,29.9,7.8,8,9.2],"duration":[6.518,2.144,1.258,1.077,1.134,11.638,19.931,0.709,0.409,10.317,1.093,1.5,13.541,1.362,0.89,11.132,0.969,0.727,7.985,0.87,1.654,15.913,6.217,10.058,6.432,4.449,1.614,0.32,1.252,12.814,1.903,4.442,6.616,2.379,3.459,11.6,4.068,2.64,0.695,2.199,3.735,1.013,3.81,10.466,3.505,3
06-16 19:52:06.196: I/ActivityManager(874): Force stopping com.mapbox.navigation.instrumentation_tests appid=10095 user=0: finished inst
06-16 19:52:06.196: D/AndroidRuntime(26204): Shutting down VM
```